specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/common-types.json

```diff
@@ -0,0 +1,826 @@
+        "x-ms-examples": {
+          "Get operation result": {
+            "$ref": "./examples/OperationResultsGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/locationName"
+          },
+          {
+            "$ref": "#/parameters/operationResultId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all of the properties of the operation result.",
+            "schema": {
+              "$ref": "#/definitions/OperationResultsDescription"
+            }
+          },
+          "default": {
+            "description": "Default error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "ServicesProperties": {
+      "description": "The properties of a service instance.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioning state.",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Creating",
+            "Accepted",
+            "Verifying",
+            "Updating",
+            "Failed",
+            "Canceled",
+            "Deprovisioned"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        },
+        "accessPolicies": {
+          "$ref": "#/definitions/ServiceAccessPoliciesInfo",
+          "description": "The access policies of the service instance."
+        },
+        "cosmosDbConfiguration": {
+          "$ref": "#/definitions/ServiceCosmosDbConfigurationInfo",
+          "description": "The settings for the Cosmos DB database backing the service."
+        },
+        "authenticationConfiguration": {
+          "$ref": "#/definitions/ServiceAuthenticationConfigurationInfo",
+          "description": "The authentication configuration for the service instance."
+        },
+        "corsConfiguration": {
+          "$ref": "#/definitions/ServiceCorsConfigurationInfo",
+          "description": "The settings for the CORS configuration of the service instance."
+        },
+        "exportConfiguration": {
+          "$ref": "#/definitions/ServiceExportConfigurationInfo",
+          "description": "The settings for the export operation of the service instance."
+        },
+        "privateEndpointConnections": {
+          "description": "The list of private endpoint connections that are set up for this resource.",
+          "type": "array",
+          "items": {
+            "$ref": "./privatelinks.json#/definitions/PrivateEndpointConnection"
```

Comment:
Here.  Please fix here.  './privatelinks.json' seems not existed.

Reply:
done.